server/methods/accounts/accounts.js

```diff
@@ -994,7 +1011,7 @@ function getDataForEmail(options) {
     user: Meteor.user(), // Account Data
     currentUserName,
     invitedUserName: name,
-    url: getEmailUrl(token)
+    url: url || getEmailUrl(token)
```

Comment:
I see how this works now, passing in a null token might be the simplest solution, though if you're also passing in a url, it doesn't seem like it should matter.